hibernate-core/src/main/java/org/hibernate/hql/internal/ast/HqlParser.java

```diff
@@ -512,11 +515,11 @@ private String toPathText(AST node) {
 	}
 
 	public Map<String, Set<String>> getTreatMap() {
-		return treatMap == null ? Collections.<String, Set<String>>emptyMap() : treatMap;
+		return treatMap == null ? Collections.emptyMap() : treatMap;
 	}
 
 	public static void panic() {
-		//overriden to avoid System.exit
+		//overridden to avoid System.exit
 		throw new QueryException( "Parser: panic" );
 	}
```

Comment:
I tried to fix some cosmetic legacy issues along the way. Not sure it is a good idea though.